docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -200,17 +200,17 @@ Type: Bool
 Default: 0
 
 
-## dns_cache_max_size
+## dns_cache_max_entries
 
-Internal DNS cache max size in bytes.
+Internal DNS cache max entries.
```

Comment:
Suggest to remove the comment in l. 207-209. It talks about the memory size in bytes but this setting is about the number of entries.

Reply:
yes, i removed it